css-positioning/fixed.css

```diff
@@ -0,0 +1,12 @@
+html {
+  font-size: 22px;
+}
+
+.page-title {
+  font-size: 3rem;
+  text-align: center;
+
+  /* your code here */
+  position: fixed;
+  right: 400px;
```

Comment:
Woah, this is a **_big_** movement. Instead, you should make the element `width: 100%` so that `text-align` can do the work of centering the text.